action.yml

```diff
@@ -23,9 +23,11 @@ inputs:
   pre-test-cmd:
     description: Extra command to invoke before ansible-test
     default:
-  python-version:
```

Comment:
This'd be a breaking API change. I think I'd prefer to be more graceful with how it's handled.

Reply:
Implemented in 6b738016f6c90c483b761be0945321fec7214d3e.